src/firmware_new/boards/frankie_v1/io/infineon_TLE5009_E1000.c

```diff
@@ -0,0 +1,70 @@
+#include "infineon_TLE5009_E1000.h"
+
+#include <stdlib.h>
+
+struct InfineonTLE5009E1000
+{
+    float A_x;
```

Comment:
Wasn't sure this was amplitude until I read the comments below. Can we rename these to be `amplitude_` and `offset_`?

Reply:
I've added comments explaining what they are in the struct. I did it like this to match the data sheet variables.